docs/integration_fetching_data.md

```diff
@@ -100,7 +101,7 @@ class MyCoordinator(DataUpdateCoordinator):
         This method will be called automatically during
         coordinator.async_config_entry_first_refresh.
         """
-        self.data = await self.my_api.load_initial_data()
+        self._initial_data = await self.my_api.load_initial_data()
```

Comment:
Maybe just take a more real life example like fetching info of the device

Reply:
done